client/grpc/tmservice/block.go

```diff
@@ -8,12 +8,12 @@ import (
 	"github.com/cosmos/cosmos-sdk/client"
 )
 
-func getBlock(clientCtx client.Context, height *int64) (*ctypes.ResultBlock, error) {
+func getBlock(ctx context.Context, clientCtx client.Context, height *int64) (*ctypes.ResultBlock, error) {
```

Comment:
in fact we are working on making `client.Context` implementing `context.Context`. But it's on hold now.